src/controllers/auth.controller.ts

```diff
@@ -16,7 +16,7 @@ export class AuthController {
   constructor(private jwtService: JWTService) {}
 
   @Post('/signup')
-  async signUp(@Body({ validate: false }) user: User, @Res() response: any) {
+  async signUp(@Body({ validate: false }) user: User, @Res() _response: any) {
```

Comment:
We should stop using `any` in these cases, `@Res() _response: User` might be better

Reply:
I think this is the normal firm for the routing-controller functions... since the response, it's not just a user